src/style/internal/mdc-variables.scss

```diff
@@ -52,6 +52,14 @@ $mdc-theme-text-secondary-on-background: rgba(0, 0, 0, 0.54);
         --lime-text-disabled-on-background-color,
         rgba(var(--contrast-1700), 0.38)
     );
+    --mdc-theme-text-primary-on-background: var(
+        --lime-text-primary-on-background-color,
+        rgba(var(--contrast-1700), 0.87)
+    );
+    --mdc-theme-text-secondary-on-background: var(
```

Comment:
Are these actually used by MDC, or are they variable names that we came up with? If they aren't defined by and used by MDC, they should not be prefixed with `--mdc-`.

If they are named the way they are because they are used by us to override the scss variable with the same name, then I think we could prefix these with `--override-mdc-` instead, so that `$mdc-theme-text-primary-on-background` becomes `--override-mdc-theme-text-primary-on-background` for example.

Reply:
(We don't have to fix that in _this_ PR, but we should fix it. (Assuming the variables aren't "from" MDC of course.))

Reply:
Right now not 100% sure, but I think they're what you could see in the CSS code applied by MDC. That's why I reused them, and make a `--lime-` version of each. So we can some day talk about how consumers can "theme" their stuff if necessary, using these variables. Today they can only specify a few things like `--lime-primary-color`. We can talk about it later.

Reply:
if you look at this page: https://github.com/material-components/material-components-web/blob/master/docs/theming.md

you'll see a bunch of such variables. For example:
![image](https://user-images.githubusercontent.com/35954987/114560416-f83bc600-9c6c-11eb-943b-7214f4052c18.png)

I think I saw them used somewhere in the original styles which are coming from MD. However, I don't remember which component it was. But that's why I decided to have them here too to ease switching to the dark mode.

But for me it's cool whichever we choose.

Reply:
Ah! Ok, then it's a different matter completely 👍